jenkins/automation/commit-tag.groovy

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env groovy
```

Comment:
Can we update the existing job instead of adding a new job.

Reply:
Thought to keep it for backup as this script do tagging based on RELEASE.INFO and new script does based on image.

Reply:
We can keep it as a backup in the branch instead of an additional script.

Reply:
took backup in branch - https://github.com/Seagate/cortx-re/tree/releaseinfo-commit-tag